bin/stacks/rpc-gateway-fallback-stack.ts

```diff
@@ -0,0 +1,147 @@
+import * as cdk from 'aws-cdk-lib'
+import { aws_cloudwatch, aws_cloudwatch_actions, aws_iam, aws_lambda, aws_lambda_nodejs } from 'aws-cdk-lib'
+import { Construct } from 'constructs'
+import path from 'path'
+import { getRpcGatewayEnabledChains } from '../../lib/rpc/ProdConfig'
+import { ComparisonOperator, MathExpression } from 'aws-cdk-lib/aws-cloudwatch'
+import { RetentionDays } from 'aws-cdk-lib/aws-logs'
+import { DynamoDBTableProps } from './routing-database-stack'
+import * as aws_dynamodb from 'aws-cdk-lib/aws-dynamodb'
+
+export interface RpcGatewayFallbackStackPros extends cdk.NestedStackProps {
+  rpcProviderHealthStateDynamoDb: aws_dynamodb.Table
+}
+
+export class RpcGatewayFallbackStack extends cdk.NestedStack {
+  constructor(scope: Construct, name: string, props: RpcGatewayFallbackStackPros) {
+    super(scope, name, props)
+
+    const lambdaRole = new aws_iam.Role(this, 'ProviderFallbackLambdaRole', {
+      assumedBy: new aws_iam.ServicePrincipal('lambda.amazonaws.com'),
+      managedPolicies: [
+        aws_iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSLambdaBasicExecutionRole'),
+        aws_iam.ManagedPolicy.fromAwsManagedPolicyName('service-role/AWSLambdaRole'),
+        aws_iam.ManagedPolicy.fromAwsManagedPolicyName('CloudWatchLambdaInsightsExecutionRolePolicy'),
+        aws_iam.ManagedPolicy.fromAwsManagedPolicyName('AWSXRayDaemonWriteAccess'),
+      ],
+    })
+
+    const rpcHealthProviderStateDynamoDB = props.rpcProviderHealthStateDynamoDb
+    rpcHealthProviderStateDynamoDB.grantReadWriteData(lambdaRole)
+
+    const region = cdk.Stack.of(this).region
+
+    const providerFallbackLambda = new aws_lambda_nodejs.NodejsFunction(this, 'ProviderFallbackLambda', {
+      role: lambdaRole,
+      runtime: aws_lambda.Runtime.NODEJS_18_X,
+      entry: path.join(__dirname, '../../lib/rpc/handler/index.ts'),
+      handler: 'fallbackHandler',
+      timeout: cdk.Duration.seconds(5),
```

Comment:
Is 5 seconds long enough? Since fallback stack is the offline path, the timeout can theoritically be longer than routing lambda (9 seconds) right?